src/bookmarklet_core/export.js

```diff
@@ -0,0 +1,9 @@
+import $ from 'jquery'
+import timesheet from '../common/js/timesheet'
+
+if (__DEV__ || document.domain === 'www.4628.jp' && $('#submit_form0').length) {
+  timesheet()
+}
+else {
+  alert('万屋一家シリーズ勤之助ver.2の「出勤簿画面」で実行してください')
+}
```

Comment:
I confirmed again what this file is doing.

If you can change the output with webpack using `if (__ DEV__)`
I thought that it would be simpler to integrate it into `src/common/js/timesheet/index.js`

What do you think?